src/devices/lifecontrol.ts

```diff
@@ -25,13 +33,19 @@ function airQuality(): ModernExtend {
 }
 
 function electricityMeterPoll(): ModernExtend {
-    const configure: Configure = async (device, coordinatorEndpoint, definition) => {
-        const endpoint = device.getEndpoint(1);
-        await reporting.bind(endpoint, coordinatorEndpoint, ['haElectricalMeasurement', 'seMetering']);
-        await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
-        await reporting.readMeteringMultiplierDivisor(endpoint);
-        await reporting.currentSummDelivered(endpoint);
-    };
+    const configure: Configure[] = [
```

Comment:
I propose to just do an `endpoint.read()` here, this is the only place where `electricityMeterPoll` is used

Reply:
I had some plans for `setupConfigureForReading`, It should be useful for attributes that act as settings and reporting them is wasteful, but without reading them first their respective expose fields will be left empty. This confuses people and creates errors in the logs sometimes